### ISSUE TITLE
Problem with new customizable layouts

### ISSUE THREAD

**Issue by rjtavares** [andfanilo/streamlit-drawable-canvas#23]
The component seems to not work with the new customizable layouts. This reproduced the problem:

```
import streamlit as st
from streamlit_drawable_canvas import st_canvas

col1, col2 = st.beta_columns(2)

with col1:
    canvas_image = st_canvas(
        fill_color = "rgba(255, 165, 0, 0.3)", 
        stroke_width = 2,
        stroke_color = '#e00',
        drawing_mode = "line",
        key = "canvas",
    )

with col2:
    options = ['U','D','R', 'L']
    lines = [st.selectbox('Select', options, key=x) for x in range(4)]

```

The page appears correctly, but keeps refreshing constantly.

**Comment by andfanilo** [comment]
Thanks for the bug report. Am pretty sure it worked during the beta something must have changed :thinking: will checkout with the team.

Best,
Fanilo

**Comment by andfanilo** [comment]
@rjtavares this should be solved in Streamlit 0.69.x. Let me know if it works on your side ;)

**Comment by rjtavares** [comment]
Perfect! Also love the new undo/redo buttons, really useful for my use case 👍
